plugins/auth-backend/src/providers/bitbucket/provider.ts

```diff
@@ -222,7 +222,7 @@ export const bitbucketUsernameSignInResolver: SignInResolver<BitbucketOAuthResul
     const claims = getEntityClaims(entity);
     const token = await ctx.tokenIssuer.issueToken({ claims });
 
-    return { id: entity.metadata.name, entity, token };
+    return { id: `user:default/${entity.metadata.name}`, entity, token };
```

Comment:
Actually, you can remove this change. The `id` is deprecated and should not be used at all, but it used to have the documented semantics that it should be "for example the metadata.name". There may be old code out there that relies on it. The important thing is that the `sub` and `ent` ones are full refs! Those fixes are good.